README.md

```diff
@@ -24,33 +24,74 @@ Then, add the format to your document options:
 ```yaml
 format:
   nature-pdf: default
-```    
+```
 
 ## Options
 
-- `journal`: the Nature subjournal
+- `natbibstyle`: the name of the natbib style for the Nature subjournal
```

Comment:
In my view the reason is because if you inspect the usage in sn-jnl.cls, you will find that these options do nothing more than set the bibliography style: 

https://github.com/cameronraysmith/nature/blob/ea663d0fb03ade55fdbb7a19d8b7c473956bb5b5/_extensions/nature/sn-jnl.cls#L107-L114

https://github.com/cameronraysmith/nature/blob/ea663d0fb03ade55fdbb7a19d8b7c473956bb5b5/_extensions/nature/sn-jnl.cls#L1641-L1723

In this sense, I found journal to be too broad in communicating what these options actually determine. This was why I decided on this name.

Reply:
Hi @cameronraysmith, thanks for the thoughts on this. After looking at some of the other Quarto templates out there, particularly those from the [Quarto team](https://github.com/quarto-journals), I think that maybe this should be a slightly different phrasing entirely.

Those templates use a nested bit of yaml, specifying it like so:
```
journal:
  cite-style: 
```

See [this search of the quarto-journals org](https://github.com/search?q=org%3Aquarto-journals%20%22journal%3A%22&type=code) to see some of those examples.

Doing that would help align the template more with other templates, while directly addressing your concern about specificity of the option. What do you think?

Reply:
That makes sense. I guess this will translate to `$journal.cite-style` in the document class definition [analogous to the few results that appear for this search](https://github.com/search?q=org%3Aquarto-journals+%22%24journal%22&type=code).

Reply:
38b409864817e00e4f3e3cb158e75b55f857e0cd